### PR TITLE
Scp0492 change apperance fix

### PR DESCRIPTION
**Pull request by Undid-Iridium**
## Description
**Describe the changes** 
https://discord.com/channels/656673194693885975/1276117397685665885/1276117397685665885
tl;dr - 0492 change appearance breaks clients before change.

longer:  changing appearance to 0492 causes clients to disconnect due to missing serialization data. When deserializing, there is a base call to a shared base deserialization function for humanoid types. 

**What is the current behavior?** (You can also link to an open issue here)
Clients crash when 0492 is used in ChangeAppearance

**What is the new behavior?** (if this is a feature change)
Doesn't crash

**Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
No

**Other information**:

<br />

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentations
<br />

## Submission checklist

- [X] I have checked the project can be compiled
- [X?] I have tested my changes and it worked as expected -> I have tested 

### Patches (if there are any changes related to Harmony patches)
- [X] I have checked no IL patching errors in the console

### Other
- [ ] Still requires more testing

https://github.com/user-attachments/assets/75e3dd31-7666-4556-ae3b-069d760882b9

### CODE REVIEW

**Review by louis1706 — CHANGES_REQUESTED** [review]
This will not work
- I don't know how you made it work but it's do not make sense lol
-

File: EXILED/Exiled.API/Extensions/MirrorExtensions.cs [hunk 1]
```diff
@@ -289,6 +289,10 @@ public static void ChangeAppearance(this Player player, RoleTypeId type, IEnumer
                     isRisky = true;
 
                 writer.WriteUShort((ushort)Mathf.Clamp(Mathf.CeilToInt(player.MaxHealth), ushort.MinValue, ushort.MaxValue));
+                writer.WriteBool(true);
+                ushort value = 0;
+                writer.WriteRelativePosition(player.RelativePosition);
+                writer.WriteUShort(value);
```
Comment (louis1706) [inline EXILED/Exiled.API/Extensions/MirrorExtensions.cs:295]:
```suggestion
```

Comment (louis1706) [reply EXILED/Exiled.API/Extensions/MirrorExtensions.cs:295]: instead put the ZombieRole before the FpcStandardRoleBase

Comment (Undid-Iridium) [reply EXILED/Exiled.API/Extensions/MirrorExtensions.cs:295]: image

Comment (Undid-Iridium) [reply EXILED/Exiled.API/Extensions/MirrorExtensions.cs:295]: image

Your suggestion is wrong as far as I can tell

Comment (Undid-Iridium) [reply EXILED/Exiled.API/Extensions/MirrorExtensions.cs:295]: Please describe in more detail what exactly you want/disagree with

Comment (Undid-Iridium) [reply EXILED/Exiled.API/Extensions/MirrorExtensions.cs:295]: I see, it's actually both - sigh, I understand

Comment (Undid-Iridium) [reply EXILED/Exiled.API/Extensions/MirrorExtensions.cs]: Also, it probably worked because of the missing data in general - if I had to guess it needed 40 bytes (example) and it had 35, so giving it 60 probably didn't matter because it only reads in the first 40. (AKA giving it the extra data after the previous condition gives the spawn data probably didn't break because it reads only a certain amount of data)

**Review by louis1706 — APPROVED** [review]

**Review by VALERA771 — APPROVED** [review]

**Review by IRacle1 — APPROVED** [review]
